src/d3a/models/market/blockchain_interface.py

```diff
@@ -99,7 +99,6 @@ def handle_blockchain_trade_event(self, offer, buyer, original_offer, residual_o
     def track_trade_event(self, trade):
 
         call_params = {
-            'id': 3,
```

Comment:
As mentioned in other thread in similar pr(https://github.com/gridsingularity/canvas-node/pull/6#discussion_r565466825), I would be in favour of using trade id here which is already unique.